### ISSUE TITLE
Launching the LSTM model as high-level without the dataset

### ISSUE THREAD

**Issue by iremustek** [Asami-1/GDP#15]
- Work on LSTM model & make sure that it works in our environment & make necessary changes

**Comment by iremustek** [comment]
a.	Problem is: https://github.com/stuarteiffert/RNN-for-Human-Activity-Recognition-using-2D-Pose-Input and https://github.com/guillaume-chevalier/LSTM-Human-Activity-Recognition  seem like they’re only working on just “1” person? That’s why I have concerns about using these? 
b.	https://github.com/imsoo/fight_detection here, they’re detecting multiple person but they don’t have the code? 
c.	Next step: searching more to find a better source? Implementing pose estimation (getting keypoints) + then use LSTM for detecting violence (or any human activity)

**Comment by Asami-1** [comment]
@IviX55 @iremustek  What we talked about yesterday is padding. We fill the input sequences with 0 to have a uniform length. It seems to be a common practice to apply masking after that, to inform the model which parts of the input as relevant and which ones should be ignored during computation (i.e the padded parts of the input).

https://www.tensorflow.org/guide/keras/masking_and_padding?hl=en  gives a good explanation on that.

**Comment by iremustek** [comment]
Completed.
